docker/dev/celery/container-start.sh

```diff
@@ -1,3 +0,0 @@
-#!/bin/sh
-cd /code && \
-celery -A evalai worker --loglevel=INFO
```

Comment:
@nikochiko we don't have to delete this

Reply:
@yashdusing This is in `docker/dev`. I don't think we will need celery in dev env?

Reply:
Nevermind this comment. I thought the dockerfile and container script was shared between the prod & dev.